docs/input.md

```diff
@@ -104,21 +104,55 @@ print(result.output)
 
 ## User-side download vs. direct file URL
 
-When you provide a URL using any of `ImageUrl`, `AudioUrl`, `VideoUrl` or `DocumentUrl`, Pydantic AI will typically send the URL directly to the model API so that the download happens on their side.
+When using one of `ImageUrl`, `AudioUrl`, `VideoUrl` or `DocumentUrl`, Pydantic AI will default to sending the URL to the model, so the file is downloaded on their side.
 
-Some model APIs do not support file URLs at all or for specific file types. In the following cases, Pydantic AI will download the file content and send it as part of the API request instead:
```

Comment:
We still need a sentence like this to explain what will happen if the table below has "Sends URL directly" as false, especially because the "false" is not explicitly stated but implied because of something like "`ImageUrl`, `AudioUrl`, `DocumentUrl` | `ImageUrl` only". It's not currently clear to the user what happens for `AudioUrl` and `DocumentUrl`